README.md

```diff
@@ -1,193 +1,266 @@
+        D(V) ~ g*Gᴾ
+    end
+    @continuous_events begin
+        [V ~ Vₘₐₓ] => [V ~ V/2]
+    end
+    kₚ*(sin(t)+1)/V, G --> Gᴾ
+    kᵢ/V, Gᴾ --> G
 end
-p     = (:c1 => 1.0, :c2 => 2.0, :c3 => 50.)
-tspan = (0.,10.)
-u0    = [:X => 5.]
-sprob = SDEProblem(rs, u0, tspan, p)
-ssol  = solve(sprob, LambaEM(), reltol=1e-3)
-plot(ssol; lw = 2, title = "Adaptive SDE: Birth-Death Process")
 ```
+Next, we can use [Latexify.jl](https://korsbo.github.io/Latexify.jl/stable/) to show the ordinary 
+differential equations associated with this model:
+```julia
+using Latexify
+latexify(cell_model; form = :ode)
+```
+In this case, we would instead like to perform stochastic simulations, so we transform our model to an SDE:
+```julia
+u0 = [:V => 0.5, :G => 1.0, :Gᴾ => 0.0]
+tspan = (0.0, 20.0)
+ps = [:Vₘₐₓ => 1.0, :g => 0.2, :kₚ => 5.0, :kᵢ => 2.0, :Ω => 0.1]
+sprob = SDEProblem(cell_model, u0, tspan, ps)
+```
+Finally, we simulate it and plot the result.
+```julia
+using StochasticDiffEq
+sol = solve(sprob, STrapezoid())
+plot(sol; xguide = "Time (au)", lw = 2)
+```
+![Elaborate SDE simulation](docs/src/assets/readme_elaborate_sde_plot.svg)
 
-![](https://user-images.githubusercontent.com/1814174/87864113-3bf9dd00-c932-11ea-8275-f903eef90b91.png)
+Some features we used here:
+- The SDE was [simulated using StochasticDiffEq.jl]. We also [scaled the SDE noise terms](@ref ref).
+- The cell volume was [modeled as a differential equation, which was coupled to the reaction network model](@ref ref).
+- The cell divisions were created by [incorporating events into the model](@ref ref).
+- The model equations were [displayed using Latexify.jl](@ref ref), and the simulation [plotted using Plots.jl](@ref ref).
 
-## Getting help
-Catalyst developers are active on the [Julia
-Discourse](https://discourse.julialang.org/), the [Julia Slack](https://julialang.slack.com) channels \#sciml-bridged and \#sciml-sysbio, and the [Julia Zulip sciml-bridged channel](https://julialang.zulipchat.com/#narrow/stream/279055-sciml-bridged).
-For bugs or feature requests [open an issue](https://github.com/SciML/Catalyst.jl/issues).
+## Getting help or getting involved
+Catalyst developers are active on the [Julia Discourse](https://discourse.julialang.org/), 
+the [Julia Slack](https://julialang.slack.com) channels \#sciml-bridged and \#sciml-sysbio, and the 
+[Julia Zulip sciml-bridged channel](https://julialang.zulipchat.com/#narrow/stream/279055-sciml-bridged). 
+For bugs or feature requests, [open an issue](https://github.com/SciML/Catalyst.jl/issues).
 
+If you are interested in participating in the development of Catalyst, or integrating your package(s) 
+with it, developer documentation can be found [here](@ref ref). Are you a student (or similar) who 
+wishes to do a Google Summer of Code (or similar) project tied to Catalyst? Information on how to get 
+involved, including good first issues to get familiar with working on the package, can be found [here](@ref ref).
```

Comment:
This last part we should remove, as we do not have any dev docs, information for people who want to get involved, or list of good issues. But if we ever get that, I think having something like this here would be nice. It is here so you can see it, then but I will remove it before we actually merge.